docker-compose.yml

```diff
@@ -23,3 +23,10 @@ services:
       - "3040"
     environment:
       ADMIN_PASSWORD: "admin"
+
+  mailcatcher:
+    restart: on-failure:10
+    image: dockage/mailcatcher:0.7.1
+    ports:
+    - "1080:1080"
+    - "1025:1025"
```

Comment:
Do you have some doc about how to use this service ? I guess you changed the `.env.local` file to test it ? Maybe it could be the default for dev environments ?